src/export/exporters/CSVExporter.js

```diff
@@ -59,6 +60,21 @@ var CSVExporter = function(form, req, res) {
         items.push({label: [path, question.value].join('.'), path: question.value});
       });
     }
+    else if (~['select', 'resource'].indexOf(component.type)) {
```

Comment:
Again, while I understand that using a Bitwise operator works here, it is hard to read and may cause some maintenance issues. Please change this to ```['select', 'resource'].indexOf(component.type) !== -1```